### ISSUE TITLE
Generating lossy access JP2s from lossless preservation masters

### ISSUE THREAD

**Issue by bitsgalore** [bitsgalore/bitsgalore.github.io#78]
<https://www.bitsgalore.org/2022/03/30/generating-lossy-access-jp2s-from-lossless-preservation-masters>

**Comment by boxerab** [comment]
Great post. Looking at the scripts, it looks like you are running OpenJPEG in single threaded mode, which would explain the large gap in performance. Adding `-threads 4` should improve performance considerably.

Also, you can grab a more recent version of Kakadu (8.05) here:

http://kakadusoftware.com/wp-content/uploads/KDU805_Demo_Apps_for_Linux-x86-64_200602.zip

from this web directory

http://kakadusoftware.com/wp-content/uploads/

**Comment by bitsgalore** [comment]
@boxerab  Thanks for the suggestion, I just re-ran all tests using the added `-threads` option for OpenJPEG. This does improve the performance, but not as much as I expected, see the updated table.

**Comment by boxerab** [comment]
@bitsgalore thanks for re-running the tests. Interesting - it is still getting crushed.
I looked up the CPU you are using,  Intel i5-6500 CPU, and there are indeed 4 threads, no hyper-threading,
so you are getting full performance.

**Comment by bitsgalore** [comment]
@boxerab Yes, I already did a quick double-check on that by trying both `-threads 4` and `-threads ALL_CPUS`. Both give me the same result, which confirms that this performance is as good as it gets on this machine.
